src/js/modules/infragistics.ui.editors.js

```diff
@@ -2398,8 +2398,8 @@
 				itemHeight = 23;
 			}
 			if (list.length < this.options.visibleItemsCount) {
-				dropdown.css("height", parseFloat(itemHeight * list.length));
-				this._listInitialHeight = parseFloat(itemHeight * list.length);
+				dropdown.css("height", parseFloat((itemHeight * list.length) + 2));
```

Comment:
Here I found code left by me (with magic numbers) - I suggest to improve it https://github.com/bazal4o/ignite-ui/tree/tpaskalev/issue_1403
If you are ok - we can keep the tests and even add more related to the item border (set to more than 1px) and even dropdownlist container border set to more than 1.